st2common/tests/unit/test_db_execution.py

```diff
@@ -188,19 +191,6 @@ def test_execution_inquiry_secrets(self):
             "supersecretvalue",
         )
 
-    def test_execution_inquiry_response_action(self):
```

Comment:
Would you please move this to `test_db_liveaction.py`?

Reply:
sounds good. will do.

Reply:
done